# A 'semantic' description of the space to be updated. The descriptions such as 'in_front_of'
# and 'on_top_of' must be specified in the model of the corresponding entity.

# Examples:
# "on_top_of table-123"        Updates table-123's pose and searches for objects on top
# "in_front_of cabinet-7"      Updates cabinet-7's pose and searches for objects in front
# "table-123"                  Updates table-123's pose
# ""                           Will update everything in view

string area_description

# what should be updated
# - main: main element of the state-update-group
# - specific: in area_description specified state-update-group member
# - dependents: all not main members of the state-update-group
string update_mode

# Padding of the world model when subtracting background for segmentation (in meters)
float32 background_padding

---
string[] new_ids      # ids of new entities
string[] updated_ids  # ids of updated entities
string[] deleted_ids  # ids of deleted entities

string error_msg      # Empty if no errors
